homeassistant/components/withings/__init__.py

```diff
@@ -255,25 +261,26 @@ async def async_subscribe_webhooks(
         # Withings will HTTP HEAD the callback_url and needs some downtime
         # between each call or there is a higher chance of failure.
         await asyncio.sleep(SUBSCRIBE_DELAY.total_seconds())
```

Comment:
It would be nice if the library handled this in the future instead of having the sleep in HA

Reply:
I will do so in a later version of the lib. I also work on new features and this PR is the last hurdle to continue